modules/apps/counter/counter-test/src/testIntegration/java/com/liferay/counter/test/CounterLocalServiceTest.java

```diff
@@ -230,7 +230,6 @@ public IncrementProcessCallable(
 				PropsKeys.COUNTER_INCREMENT + "." + _counterName, "1");
 
 			System.setProperty("catalina.base", ".");
-			System.setProperty("external-properties", "portal-test.properties");
```

Comment:
@yuhai @dantewang 
You can not just simply remove the properties reading, you are missing the properties in portal-test.properties, you should inline those properties or you have checked all the properties in portal-test.properties and you are sure no one would affect this test. But according to the history, it seems at least "scheduler.enabled" matters.

Tina.

Reply:
@tinatian We will recheck the properties. But any property related to what is now in OSGi module shouldn't be needed, because the sub-processes doesn't init module framework:

```
InitUtil.initWithSpring(
	Arrays.asList(
		"META-INF/base-spring.xml", "META-INF/counter-spring.xml"),
	false, true);
```

So `scheduler.enabled=false` in portal-test.properties will not affect at least the sub-processes.